client/src/pages/Step/index.js

```diff
@@ -46,82 +82,195 @@ function Step() {
+              {step.pageTitle || step.title}
+            </T.H1>
+            <T.P color="neutralDark" mbT="5">
+              {step.pageDescription || step.description}
+            </T.P>
+          </Col>
+          {step.topTip && (
+            <Row inner>
               <Col w={[4, 12, 6]}>
-                <Button
-                  variant="primary"
-                  text={t(`${step.name}.externalButtonTitle`, lang)}
-                  to={n.EXTERNAL[step.externalButtonLink]}
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  external
-                />
+                <Tips tips={[step.topTip]} ml="5" mlT="0" />
               </Col>
             </Row>
           )}
-          <T.P isSmall weight="bold" mb="5">
-            {t('informationYouWillNeed', lang)}
-          </T.P>
-          <Row inner>
-            {step.checkListItems.map((item) => (
-              <Col w={[4, 12, desktopColWidth]}>
-                <Checkbox
-                  key={item.value}
-                  checked={item.isChecked}
-                  handleChange={() => checkUncheckItem(step.name, item.value)}
-                  label={t(`${step.name}.checkListItems.${item.value}`, lang)}
-                  mb="5"
+        </Row>
+
+        {step.howLongDoesItTake && (
+          <Row>
+            <Col w={[4, 6, 6]}>
+              <S.SectionHeader mb="2">
+                <Icon
+                  icon="time"
+                  width={24}
+                  height={24}
+                  color="primaryMain"
+                  mr="2"
                 />
-              </Col>
-            ))}
+                <T.H2 color="neutralMain">How long does it take?</T.H2>
+              </S.SectionHeader>
+              <T.P color="neutralDark">
+                {step.howLongDoesItTake.timeRangeText}
+              </T.P>
+            </Col>
           </Row>
-        </div>
-
-        <Row
-          inner
-          style={{
-            width: '100%',
-            minHeight: '58px',
-            alignSelf: 'flex-end',
-          }}
-        >
-          {step.isCompleted ? (
-            <Col w={[4, 12, 6]} mt="6" mb="7">
-              <Button
-                variant="secondary"
-                text={t('nextStep', lang)}
-                to={n.GENERAL.HOME}
-                handleClick={() => {
-                  setJustCompletedId(step.id);
-                }}
+        )}
+
+        <Row mt="8" mtM="7">
+          <Col w={[4, 12, 12]}>
+            <S.SectionHeader mb="5">
+              <Icon
+                icon="time"
+                width={24}
+                height={24}
+                color="primaryMain"
+                mr="2"
               />
+              <T.H2 color="neutralMain">Things you'll need</T.H2>
+            </S.SectionHeader>
+            <Row inner>
+              {checklist.thingsYouWillNeed?.length > 0 ? (
+                checklist.thingsYouWillNeed.map((item, index) => (
```

Comment:
```suggestion
                {checklist?.thingsYouWillNeed?.map((item, index) => (
```

Reply:
hey Ramy this is part of a ternary so we do need the first bit as otherwise we render something else

Reply:
ah sorry ok